dts/bindings/input/nintendo,nunchuk.yaml

```
@@ -0,0 +1,14 @@
+# Copyright (c) 2024 Bootlin
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nintendo Nunchuk joystick through I2C
+
+compatible: "nintendo,nunchuk"
+
+include: "i2c-device.yaml"
```

```suggestion
include: i2c-device.yaml
```

(not too sure why but most entries don't use quotes for single includes)